src/index.js

```diff
@@ -1,38 +1,57 @@
 'use strict'
 
-var path = require('path')
+var fs = require('fs')
 var goenv = require('go-platform')
-var version = require('./../package.json').version
-var request = require('request')
 var gunzip = require('gunzip-maybe')
+var path = require('path')
+var request = require('request')
 var tarFS = require('tar-fs')
+var unzip = require('unzip')
+var version = require('./../package.json').version
 
 module.exports = function (callback) {
   callback = callback || noop
+
+  var fileName, fileExtension, url, installPath, isUnixy
+
   checkPlatform(goenv) // make sure we can do this.
 
   // hacky hack hack to work around unpublishability
   version = version.replace(/-[0-9]+/, '')
 
-  var filename = 'ipfs_v' + version + '_' + goenv.GOOS + '-' + goenv.GOARCH + '.tar.gz'
-  var url = 'http://dist.ipfs.io/go-ipfs/v' + version + '/go-' + filename
+  isUnixy ? fileExtension = '.tar.gz' : fileExtension = '.zip'
+
+  fileName = 'ipfs_v' + version + '_' + goenv.GOOS + '-' + goenv.GOARCH + fileExtension
+  url = 'http://dist.ipfs.io/go-ipfs/v' + version + '/go-' + fileName
 
-  var installPath = path.resolve(__dirname, '..')
+  installPath = path.resolve(__dirname, '..')
 
-  request
+  if (isUnixy) {
```

Comment:
Let's call it `isWindows` instead and make it the function instead. Then break it to a separate file